### DISCUSSION TITLE
Why is Clippy wrong here?

### DISCUSSION THREAD

**Discussion by rafrafek** [rust-lang/rust-clippy#13152] [Q&A] [answered]
Why is Clippy wrong here?

https://github.com/denoland/deno/pull/24593/commits/3dbd158d791713c5ca52b7a7630a2f593c7e7626

```rust
// clippy is wrong here
#[allow(clippy::declare_interior_mutable_const)]
const STAR_STAR: HeaderValue = HeaderValue::from_static("*/*");
```

**Comment by Alexendoo** [comment, marked as answer]
https://github.com/rust-lang/rust-clippy/pull/12691 has not yet hit stable, but it's fixed in nightly

**Reply by rafrafek** [reply]
Great, thanks!

**Reply by AsrizalBasri** [reply]
thanks
